bentoml/frameworks/spark_mllib.py

```diff
@@ -0,0 +1,128 @@
+import importlib
+import json
+import logging
+import os
+
+from bentoml.exceptions import BentoMLException, MissingDependencyException
+from bentoml.service import BentoServiceArtifact, BentoServiceEnv
+
+
+class PySparkModelArtifact(BentoServiceArtifact):
+    """
+    PySparkModelArtifact allows you to use the spark.ml and spark.mllib APIs with Bento
```

Comment:
Could you also add the `Parameters` list in docstring and explain the expected parameter for `__init__` and what are their types, and what exception this could raise? Would be great to also add docstring for pack and list documentation for its parameters